File: Questions/src/Recursion/sumOfArray.java
package Recursion;

public class sumOfArray {
//    int arr[];
    int sum=0;
    public int result(int arr[],int size){
        if(size<0){
            return 0;
        }
        else{
            sum+=arr[size-1];
            result(arr, size-1);
        }
        return sum;
    }
    public static void main(String[]args){
        sumOfArray ob=new sumOfArray();
        int arr[]={1,2,3,4,5};
        int res=ob.result(arr, arr.length);
        System.out.println("Result: "+res);
    }
}

File: Questions/src/C_Program/sumOfArray.c
#include<stdio.h>
int result(){
    int arr[100],n,sum=0;
    printf("Enter the Size of the  Array\n");
    scanf("%d",&n);
    printf("Enter the Value in the Array\n");
    for(int i=0;i<n;i++){
        scanf("%d",&arr[i]);
    }
    for(int i=0;i<n;i++){
        sum+=arr[i];
    }
    printf("Sum: %d\n",sum);
}
void main(){
    int n;
    printf("How many time you want to call the Function\n");
    scanf("%d",&n);
    for(int i=1;i<=n;i++){
        printf("Enter the Value in %d\n",i);
        result();
    }
}